jenkins/build-opm-module.sh

```diff
@@ -130,8 +130,18 @@ function clone_module {
   else
     git remote add origin https://github.com/OPM/$1
   fi
-  git fetch --depth 1 origin $2:branch_to_build
-  git checkout branch_to_build
+  # Complication due to github slow merge ref updates
+  if [ "$2" == "master" ]
+  then
+    git fetch --depth 1 origin $2:branch_to_build
+    git checkout branch_to_build
+  else
+    git fetch --depth 1 origin master
+    git fetch --depth 1 origin $2:branch_to_build
+    git checkout master
+    git merge branch_to_build
+    test $? -eq 0 || exit 1
```

Comment:
Is there a reason not to just use:
```sh
#/bin/bash
set -e
```
to capture *all* errors?

Reply:
not really in this case. but i like to have control over my exit codes in general..